tests/functional/test_ems_end2end.py

```diff
@@ -600,11 +612,36 @@ def validate_test_server(cls):
         cls.assert_test(lambda: resp.json.get("configuration") == WEAVER_CONFIGURATION_EMS,
                         message="weaver must be configured as EMS.")
 
+    @pytest.mark.skip(reason="Workflow not working anymore. IO to be repaired.")
     def test_workflow_wps1_requirements(self):
         self.workflow_runner(self.PROCESS_WORKFLOW_SUBSET_ICE_DAYS,
                              [self.PROCESS_SUBSET_BBOX_ID, self.PROCESS_ICE_DAYS_ID],
                              log_full_trace=True)
 
+    def test_workflow_subset_picker(self):
+        self.workflow_runner(self.PROCESS_WORKFLOW_SUBSET_PICKER,
+                             [self.PROCESS_SUBSET_BBOX_ID],
+                             log_full_trace=True)
+
+    @pytest.mark.skip(reason="Workflow not working anymore. IO to be repaired.")
+    def test_workflow_llnl_subset_esgf(self):
+        self.workflow_runner(self.PROCESS_WORKFLOW_SUBSETLLNL_SUBSETCRIM,
+                             [self.PROCESS_SUBSET_ESGF, self.PROCESS_SUBSET_BBOX_ID],
+                             log_full_trace=True)
+
+    @pytest.mark.skip(reason="Workflow not working anymore. IO to be repaired.")
+    def test_workflow_esgf_requirements(self):
+        self.workflow_runner(self.PROCESS_WORKFLOW_SUBSETNASAESGF_SUBSETCRIM,
+                             [self.PROCESS_SUBSET_NASAESGF, self.PROCESS_SUBSET_BBOX_ID],
+                             log_full_trace=True)
+
+    @pytest.mark.skip(reason="Workflow not working anymore. IO to be repaired.")
+    def test_workflow_file_to_string_array(self):
+        self.workflow_runner(self.PROCESS_WORKFLOW_FILE_TO_SUBSETCRIM,
+                             [self.PROCESS_SUBSET_BBOX_ID],
+                             log_full_trace=True)
+
+    @pytest.mark.skip(reason="Workflow not working anymore. IO to be repaired.")
```

Comment:
idem, avoid skiping

Reply:
ok, changed to `xfail` then, since they don't currently work